src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/ExceptionRegisterKeyStoreProvider.cs

```diff
@@ -4,69 +4,126 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Reflection;
 using Xunit;
 
 namespace Microsoft.Data.SqlClient.Tests.AlwaysEncryptedTests
 {
     public class ExceptionRegisterKeyStoreProvider
     {
+        private SqlConnection connection = new SqlConnection();
+
+        IDictionary<string, SqlColumnEncryptionKeyStoreProvider> singleKeyStoreProvider =
+            new Dictionary<string, SqlColumnEncryptionKeyStoreProvider>()
+            {
+                { "DummyProvider1", new DummyKeyStoreProvider() }
```

Comment:
NIT: Using a constant for strings like this that you reuse in another method is a good practice.